vue/components/ui/organisms/listing/listing.vue

```diff
@@ -246,6 +247,9 @@ export const Listing = {
         searchWidth: {
             type: Number,
             default: 304
+        },
+        lineupNrColumns: {
+            type: Number

```

Comment:
Either `required: true` or `default: something`. In this case let's go with `default: null`, delegating to the lineup whatever the default should be.

Reply:
Setting it `default: null `will make the prop in lineup be `null`. That's why I didn't added a default to the "lineupNrColumns" prop, as I wanted the default value of "nrColumns" in "lineup" component be controlled by itself.